cmd/nerdctl/network/network_create_linux_test.go

```diff
@@ -110,3 +111,97 @@ func TestNetworkCreate(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+func TestNetworkCreateICC(t *testing.T) {
+	testCase := nerdtest.Setup()
+
+	testCase.Require = require.All(
+		require.Linux,
+		nerdtest.Rootful,
```

Comment:
Why?

Reply:
I have removed the require for 'rootful'. Test should pass for both rootful and rootless.